### DISCUSSION TITLE
如何在useModel中使用国际化

### DISCUSSION THREAD

**Discussion by a1352723557** [umijs/umi#10695] [Q&A]
比如以下useModel文件代码： 
`
import {useIntl} from 'umi'
export default function usePageStore(){
  const {formatMessage} = useIntl();
  return {
name: formatMessage({id: 'name', defaultMessage:'Hello Word'})
}
}
`

**Comment by fz6m** [comment]
给一个最小复现看看。

**Reply by a1352723557** [reply]
可以看下这个简单实例:  https://github.com/a1352723557/test/blob/main/src/models/global.ts
启动报错如下：
<img width="637" alt="test" src="https://user-images.githubusercontent.com/41726893/223598124-3a332318-678a-4509-b6a8-205ac40c4be2.png">

**Reply by a1352723557** [reply]
简单业务场景就是在model层调用接口，接口响应的时候，需要做message提示。

**Reply by fz6m** [reply]
https://github.com/umijs/umi/issues/10702

**Comment by a1352723557** [comment]
同类问题： https://github.com/umijs/umi/issues/10702
